os/include/tinyara/lcd/st7785.h

```diff
@@ -59,6 +59,22 @@
 
 #define LCDC_IMG_BUF_SIZE               LCDC_IMG_BUF_ALIGNED64B(LCD_XRES * LCD_YRES * 2)
 
+#define ST7785_COMMON_INIT         {0xB0, 2, {0x00, 0x10}},	/* RAM control */                     \
+        {0xB2, 10, {0x00, 0x0C, 0x00, 0x0C, 0x00, 0x00, 0x00, 0x33, 0x00, 0x33}}, /* Porch Setting */ \
+        {0xB7, 2, {0x00, 0x51}},	/* Gate Control */                                            \
+        {0xBB, 2, {0x00, 0x22}},	/* VCOMS Setting */                                           \
+        {0xC0, 2, {0x00, 0x2C}},	/* LCM Control */                                             \
+        {0xC2, 2, {0x00, 0x01}},	/* VRH Command Enable */                                      \
+        {0xC3, 2, {0x00, 0x13}},	/* VRH Set */                                                 \
+        {0xC6, 2, {0x00, 0x0F}},	/* Frame Rate Control in Normal Mode */                       \
+        {0xD0, 2, {0x00, 0xA7}},	/* Power Control 1 */                                         \
+        {0xD0, 4, {0x00, 0xA4, 0x00, 0xA1}}, /* Power Control 1 */                                    \
+        {0xE0, 28, {0x00, 0xF0, 0x00, 0x00, 0x00, 0x04, 0x00, 0x03, 0x00, 0x05, 0x00, 0x04, 0x00, 0x2E, 0x00, 0x44, 0x00, 0x45, 0x00, 0x39, 0x00, 0x14, 0x00, 0x14, 0x00, 0x2D, 0x00, 0x35}}, /* Positive Voltage Gamma Control */ \
+        {0xE1, 28, {0x00, 0xF0, 0x00, 0x0A, 0x00, 0x0E, 0x00, 0x0F, 0x00, 0x0B, 0x00, 0x26, 0x00, 0x2E, 0x00, 0x43, 0x00, 0x45, 0x00, 0x36, 0x00, 0x12, 0x00, 0x12, 0x00, 0x2A, 0x00, 0x32}}, /* Negative Voltage Gamma Control */ \
+        {0x2A, 8, {0x00, 0x00, 0x00, 0x00, 0x00, LCD_CASET1, 0x00, LCD_CASET2}},	/* Column Address Set */ \
+        {0x2B, 8, {0x00, 0x00, 0x00, 0x00, 0x00, LCD_RASET1, 0x00, LCD_RASET2}},	/* Row Address Set */    \
+        {0x21, 0, {0x00}},	/* Display Inversion On */
+
 static const lcm_setting_table_t lcd_init_cmd_g[] = {
```

Comment:
This type lcd can not be used. 
Could you please make configuration and make disabling it?
The lcd table has 0.6k static memory size, so for reduce memory, exculding it is better in build

Reply:
And This type lcd is also AVD vender lcd.

we have two type lcd
and one of lcd has two type vender. 

How about below

#if defined(CONFIG_LCD_ST7785_PANEL_A)
static const lcm_setting_table_t lcd_init_cmd_st7785_panelA_avd[]
#elif (CONFIG_LCD_ST7785_PANEL_B)
static const lcm_setting_table_t lcd_init_cmd_st7785_panelB_avd[]
static const lcm_setting_table_t lcd_init_cmd_st7785_panelB_holitech[]
#endif

the naming just is suggestion. Please make collectly name